feat: add copy to clipboard in search for icons

nanthakumaran-s opened the pull request:
## Fixes Issue
PR for issue #2839 
Added `react-toastify` and copy on clipboard option
Closes #2839 

## Changes proposed

- components/IconCard.js
- package.json

## Check List (Check all the applicable boxes) <!-- Follow the above conventions to check the box -->

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots

https://user-images.githubusercontent.com/59391441/211196050-ed6ffc63-2f96-4116-94b7-7e724fa533f2.mov



<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2869"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>

eddiejaoude commented on package.json:
```diff
@@ -22,6 +22,7 @@
     "react-dom": "^18.2.0",
     "react-icons": "^4.6.0",
     "react-markdown": "^8.0.3",
+    "react-toastify": "^9.1.1",
```
We are using tailwind, I think we don't need to introduce new libraries for this. Here are some prebuilt components from Tailwind - let me know which one you want to use and I can share their code (I bought it to use on this project)

https://tailwindui.com/components/application-ui/overlays/notifications

nanthakumaran-s replied:
Simple Notification

<img width="1339" alt="image" src="https://user-images.githubusercontent.com/59391441/211564521-f23f399b-ed1d-4769-87bb-023690b22db1.png">

eddiejaoude replied:
Thanks @nanthakumaran-s 👍 

```js
import { Fragment, useState } from 'react'
import { Transition } from '@headlessui/react'
import { CheckCircleIcon } from '@heroicons/react/24/outline'
import { XMarkIcon } from '@heroicons/react/20/solid'

export default function Example() {
  const [show, setShow] = useState(true)

  return (
    <>
      {/* Global notification live region, render this permanently at the end of the document */}
      <div
        aria-live="assertive"
        className="pointer-events-none fixed inset-0 flex items-end px-4 py-6 sm:items-start sm:p-6"
      >
        <div className="flex w-full flex-col items-center space-y-4 sm:items-end">
          {/* Notification panel, dynamically insert this into the live region when it needs to be displayed */}
          <Transition
            show={show}
            as={Fragment}
            enter="transform ease-out duration-300 transition"
            enterFrom="translate-y-2 opacity-0 sm:translate-y-0 sm:translate-x-2"
            enterTo="translate-y-0 opacity-100 sm:translate-x-0"
            leave="transition ease-in duration-100"
            leaveFrom="opacity-100"
            leaveTo="opacity-0"
          >
            <div className="pointer-events-auto w-full max-w-sm overflow-hidden rounded-lg bg-white shadow-lg ring-1 ring-black ring-opacity-5">
              <div className="p-4">
                <div className="flex items-start">
                  <div className="flex-shrink-0">
                    <CheckCircleIcon className="h-6 w-6 text-green-400" aria-hidden="true" />
                  </div>
                  <div className="ml-3 w-0 flex-1 pt-0.5">
                    <p className="text-sm font-medium text-gray-900">Successfully saved!</p>
                    <p className="mt-1 text-sm text-gray-500">Anyone with a link can now view this file.</p>
                  </div>
                  <div className="ml-4 flex flex-shrink-0">
                    <button
                      type="button"
                      className="inline-flex rounded-md bg-white text-gray-400 hover:text-gray-500 focus:outline-none focus:ring-2 focus:ring-indigo-500 focus:ring-offset-2"
                      onClick={() => {
                        setShow(false)
                      }}
                    >
                      <span className="sr-only">Close</span>
                      <XMarkIcon className="h-5 w-5" aria-hidden="true" />
                    </button>
                  </div>
                </div>
              </div>
            </div>
          </Transition>
        </div>
      </div>
    </>
  )
}
```

nanthakumaran-s commented:
Removed `react-toastify`
Added tailwind notification component

https://user-images.githubusercontent.com/59391441/211967959-c568ffbf-4158-4815-8bb7-b117764e6b14.mov

eddiejaoude commented:
Looks great 👍 

But please can you revert changes to the `package` and `package-lock` files. 

You can use `git checkout main -- package.json`

eddiejaoude reviewed (changes requested):
Revert changes to package and lock file

nanthakumaran-s commented:
Reverted the changes in `package.json` and `package-lock.json`

eddiejaoude reviewed (approved):
Looks good 👍
